jwst/fringe/fringe_step.py

```diff
@@ -13,7 +13,7 @@ class FringeStep(Step):
     reference_file_types = ['fringe']
 
     def process(self, input):
-        with datamodels.ImageModel(input) as input_model:
+        with datamodels.open(input) as input_model:
```

Comment:
Theoretically the fringe step can only ever be applied to an input in the form of an IFUImageModel, so should we hardwire the open to use that particular data model, e.g. datamodels.IFUImageModel? I know that the generic .open() function should automatically use the model type set in DATAMODL, but what if someone feeds in a generic, simulated image or something that doesn't have the DATAMODL keyword. We want to know right up front if the input isn't compatible with an IFUImageModel.